### PR TITLE
chore: move style linters to the `style` namespace

### PR DESCRIPTION
**Pull request by grunweg**
- [x] depends on: #15017

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: lakefile.lean
Comment (joneugster) [inline lakefile.lean:1]: I think this needs some `linter.style.` added, right?

**Comment by grunweg** [comment]
@joneugster I have addressed your comment. Do you mind taking another look?

**Comment by joneugster** [comment]
Looks good, but now you've created #16212 let's wait for that to be merged and resolve the merge conflicts here.

**Comment by grunweg** [comment]
Let's land #15017 first; or I can split out another chunk of these changes.

**Comment by grunweg** [comment]
Actually, #16265 already includes everything from this PR: let me close this one instead.

**Pull request closed without merging**